lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js

```diff
@@ -127,6 +130,14 @@ testRule({
 			endLine: 1,
 			endColumn: 10,
 		},
+		{
+			code: 'a:modal { }',
+			message: messages.rejected(':modal'),
+			line: 1,
+			column: 2,
+			endLine: 1,
+			endColumn: 10,
+		},
```

Comment:
Hm, it's unclear to me why this test case is here; we *don't* want to reject `:modal` since it is a known pseudo-class. Could you remove this test case?

```suggestion
```